src/ceph.in

```diff
@@ -639,7 +639,7 @@ def main():
         return 0
     elif sockpath:
         try:
-            print(admin_socket(sockpath, childargs, format))
+            raw_write(admin_socket(sockpath, childargs, format))
```

Comment:
Looks like the build failure [1] is because this function is not defined anywhere.

[1] NameError: global name 'raw_write' is not defined